tools/testing/target_determination/heuristics/utils.py

```diff
@@ -44,21 +44,41 @@ def query_changed_files() -> List[str]:
     return lines
 
 
-def get_correlated_tests(file: Union[str, Path]) -> List[str]:
+def normalize_ratings(ratings: Dict[str, float], max_value: float) -> Dict[str, float]:
```

Comment:
Could you add a short comment explaining how this is supposed to be used, especially the role max_value plays?